apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -12,6 +12,7 @@ import * as os from 'os';
 import * as path from 'path';
 import * as semver from 'semver';
 import * as tar from 'tar';
+import { isEmpty } from 'lodash';
```

Comment:
Remove. I think `Object.keys` or `Object.entries` length check should be just as valid.

Reply:
Fixed